src/pollingLink.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  Operation,
+  NextLink,
+  FetchResult,
+} from './types';
+
+import * as Observable from 'zen-observable';
+
+import {
+  ApolloLink,
+} from './link';
+
+
+export default class PollingLink extends ApolloLink {
+
+  private pollInterval: number;
+  private timer;
+
+  constructor(pollInterval?: number) {
+    super();
+    this.pollInterval = pollInterval || 0;
```

Comment:
@evanshauser are you open to it being a function that given the operation, can determine the interval?